### PR TITLE
improved inserted html

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

File: addon/components/roadsigns-modal.js [hunk 1]
```diff
@@ -78,6 +79,21 @@ export default class RoadsignRegulationCard extends Component {
...
   insertHtml(html) {
-    this.args.controller.executeCommand('insert-html', html);
+    const wrappedHtml = `
+      <div property="eli:has_part" typeof="besluit:Artikel" resource="http://data.lblod.info/artikels/${uuid()}">
+        <div property="eli:number" datatype="xsd:string">Artikel <span class="mark-highlight-manual">nummer</span></div>
+        <span style="display:none;" property="eli:language" resource="http://publications.europa.eu/resource/authority/language/NLD" typeof="skos:Concept">&nbsp;</span>
+        <div property="prov:value" datatype="xsd:string">
+          <span class="mark-highlight-manual">Voer inhoud in</span>
+        </div>
+        <div property="mobiliteit:heeftVerkeersmaatregel" typeof="mobiliteit:Mobiliteitsmaatregel" resource="http://data.lblod.info/mobiliteitsmaatregel/${uuid()}">
```
Comment (nvdk) [inline addon/components/roadsigns-modal.js:115]: given that mobiliteit is a specific vocabulary I think it's best the specify the prefix in this snippet

File: addon/components/roadsigns-modal.js [hunk 2]
```diff
@@ -72,6 +78,7 @@ export default class RoadsignRegulationCard extends Component {
       this.endpoint
     );
     this.tableData = signs;
+    console.log(signs)
```
Comment (nvdk) [inline addon/components/roadsigns-modal.js]: log statement can go :)

**Comment by lagartoverde** [comment]
yep, I'm going to merge all the changes about the mappings here, I will fix now your comments so I don't forget about them :)

**Comment by lagartoverde** [comment]
This PR has evolved into "just made it work as it should" now

**Pull request merged**